PubNubSwiftConsole/Classes/Components/ConsoleViewController.swift

```diff
@@ -568,6 +581,22 @@ public class ConsoleViewController: CollectionViewController, CollectionViewCont
         }
     }
     
+    public func client(client: PubNub, didReceivePresenceEvent event: PNPresenceEventResult) {
+        collectionView?.performBatchUpdates({
+            let receivedPresenceEvent = Presence(itemType: ConsoleItemType.PresenceEvent, event: event)
+            guard let currentDataSource = self.dataSource as? ConsoleDataSource else {
+                return
+            }
+            // the indexPath is the same for both calls
+            let messageIndexPath = currentDataSource.push(receivedPresenceEvent, consoleSection: .PresenceEvents)
```

Comment:
probably nitpicking, but let's name this to be presenceEventIndexPath, just because I can tell this is copy/pasted